libs/prelude/Prelude/Interfaces.idr

```diff
@@ -176,10 +176,16 @@ interface Applicative m => Monad m where
 
 %allow_overloads (>>=)
 
+||| Sequencing of effectful composition
 public export
 (>>) : (Monad m) => m a -> m b -> m b
 a >> b = a >>= \_ => b
 
+||| Kleisli composition of effectful computations
+public export
+(>>>) : Monad m => (a -> m b) -> (b -> m c) -> (a -> m c)
```

Comment:
Why not `>=>`?

Reply:
I was reading the Freer paper, that's what they used & I assumed it was standard.
Looking at hoogle, `>=>` seems to have the more precise type (and `>>>` to
be used for arrows in a category which Kleisli arrows are).

Reply:
Ha! As it turns out we had some of these in `Control.Monad.Syntax`.
I've moved them all to `Prelude.Interfaces`.